### ISSUE TITLE
Nothing is happening

### ISSUE THREAD

**Issue by jalla2000** [funkytechmonkey/razer-leviathan#4]
I'm using a USB stick of type HyperX Fury 16GB USB 3.0 formatted with FAT32 filesystem and moved the beethoven file to the stick. Formatted using Windows 10 Home. Safely ejected stick. Unplugged all cables. Plugged in USB stick. Plugged in power. Nothing happened (No LEDs lighting up or anything).

What could I be doing wrong? Compatibility problem with USB3.0 sticks?

**Comment by zaggino** [comment]
@jalla2000 I have just flashed this using 32GB USB 3.0 stick from Lexar and it flashed fine. Did you copy only the `beethoven11_v1.0.0.0.mdu` file and nothing else on the disk? Make sure it's the only file on the stick and it's not in any directory.

**Comment by NickeManarin** [comment]
Yeah, not working with mine too. Kingston Travel 2.0 4GB, formatted as fat32, only the Beethoven file.

**Comment by cyrus104** [comment]
Not working for me either.

**Comment by Vauzgh** [comment]
Not working

**Comment by vlad610** [comment]
Not working, nothing happens after I plug in Power cable, no LEDs lighting up or anything...

**Comment by DillonN** [comment]
Try different USB sticks. I had to try 4 before I found one that worked

**Comment by NickeManarin** [comment]
I tested it again, now with a Sandisk Cruzer Blade 32GB (USB 2.0) and it worked.

During the flash process (the leds will light one by one, repeating until it ends):
![1](https://user-images.githubusercontent.com/14798947/94218779-9deb3b80-febb-11ea-9895-42ee7230a1d2.jpeg)

Finished:
![2](https://user-images.githubusercontent.com/14798947/94218784-9e83d200-febb-11ea-8236-4bb4f3fca996.jpeg)

By the way, if you put a bad file in the USB, the Leviathan will light up the mute button (clicking on it will not turn off the mute).

**Comment by Billard0522** [comment]
> I tested it again, now with a Sandisk Cruzer Blade 32GB (USB 2.0) and it worked.
> 
> During the flash process (the leds will light one by one, repeating until it ends): ![1](https://user-images.githubusercontent.com/14798947/94218779-9deb3b80-febb-11ea-9895-42ee7230a1d2.jpeg)
> 
> Finished: ![2](https://user-images.githubusercontent.com/14798947/94218784-9e83d200-febb-11ea-8236-4bb4f3fca996.jpeg)
> 
> By the way, if you put a bad file in the USB, the Leviathan will light up the mute button (clicking on it will not turn off the mute).

After you update the firmware, do you have any issue on the speaker?

**Comment by DaddyFrosty** [comment]
worked perfectly for me after I switched from a modern USB 3.0 stick to a old USB 2.0 stick.

**Comment by dWoolridge** [comment]
I'm late to the party here, but I had the same problem.

Somehow, when I downloaded the beethoven11_v1.0,0,0.mdu the first time it was only like 126kB.  When I downloaded it a second time, it was 826kB.  Not sure how that happened, but the right sized file (826kB) worked.
